app/src/Book.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace App;
+/**
+ * Book is a DB entity, with a constructor to create itself. The constructor has a default of null and can be passed an individual book id.
+ * @package App
+ */
+class Book
+{
+    public $db;
+    public $id;
+    public $title;
+    public $price;
+    public $description;
+    public $image;
+
+    /**
+     * Book constructor.
+     * @param int $id (book id)
```

Comment:
wrong order, first param is the db, second is the book id. need to say they are optional too